src/client.ts

```diff
@@ -882,10 +881,8 @@ export class MatrixClient extends EventEmitter {
             // Ensure the unread counts are kept up to date if the event is encrypted
             // We also want to make sure that the notification count goes up if we already
             // have encrypted events to avoid other code from resetting 'highlight' to zero.
-            const oldHighlight = oldActions && oldActions.tweaks
-                ? !!oldActions.tweaks.highlight : false;
-            const newHighlight = actions && actions.tweaks
-                ? !!actions.tweaks.highlight : false;
+            const oldHighlight = !!oldActions?.tweaks?.highlight ?? false;
+            const newHighlight = !!actions?.tweaks?.highlight ?? false;
```

Comment:
This transpiles to `(_b = !!((_a = oldActions === null || oldActions === void 0 ? void 0 : oldActions.tweaks) === null || _a === void 0 ? void 0 : _a.highlight)) !== null && _b !== void 0 ? _b : false;`

Given the `!!` it'll never be null/undefined